src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/DialogScript

```diff
@@ -699,6 +711,7 @@
             default { "0" }
             parmtag { "script_callback" "hou.phm().callback(kwargs)" }
             parmtag { "script_callback_language" "python" }
+            help    "Parse the input and output file names and directories based on the current ROP graph."
```

Comment:
I think we could clarify a bit more what parsing means. Maybe something like, "Discover and attach related assets to the job."